tensorflow/tools/ci_build/windows/bazel/common_env.sh

```diff
@@ -44,6 +44,6 @@ export PATH="/c/Program Files/Anaconda3:$PATH"
 export PATH="/c/Program Files/Anaconda3/Scripts:$PATH"
 
 # Add Cuda and Cudnn dll directories into PATH
-export PATH="/c/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v8.0/bin:$PATH"
-export PATH="/c/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v8.0/extras/CUPTI/libx64:$PATH"
+export PATH="/c/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v9.0/bin:$PATH"
+export PATH="/c/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v9.0/extras/CUPTI/libx64:$PATH"
```

Comment:
@gunan @av8ramit - I'm guessing these weren't picked up in the 1.5 build since the Windows pip packages aren't built by bazel? If so, please do remember to update any CUDA-checking scripts so that when 1.6 rolls out (and we switch to CUDA9.1 or something), then these get updated too :)

Reply:
+1

Reply:
@angersson might be worth checking if any of these merit going into 1.5. I'll make a note for 1.6